### PR TITLE
Remove usage of skins script

### PR DESCRIPTION
**Pull request by jensens**
see https://github.com/plone/Products.CMFPlone/issues/3515
Transitive fix for https://github.com/plone/Products.CMFPlone/pull/3960 too

### CODE REVIEW

File: plone/app/content/browser/selection.py [hunk 1]
```diff
@@ -100,9 +100,12 @@ def get_selectable_items(self):
         view_types = registry.get("plone.types_use_view_action_in_listings", [])
         default_page_types = registry.get("plone.default_page_types", [])
         portal_types = getToolByName(self.context, "portal_types")
+        portal_catalog = getToolByName(self.context, "portal_catalog")
 
         results = []
-        for brain in context.getFolderContents():
+        for brain in portal_catalog(
+            path={"query": "/".join(context.getPhysicalPath()), "depth": 1}
```
Comment (davisagli) [inline plone/app/content/browser/selection.py:107]: fwiw, you could use context.absolute_url_path() here

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: plone/app/content/browser/selection.py (hunk 1 above)
Comment (davisagli) [reply plone/app/content/browser/selection.py:107]: The getFolderContents script also sorts by getObjPositionInParent by default

Comment (jensens) [reply plone/app/content/browser/selection.py]: thx, I updated it.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**